src/app/clusters/scenes-server/SceneTable.h

```diff
@@ -203,8 +211,7 @@ class SceneTable
 
         void Clear()
         {
-            SetName(CharSpan());
-
+            memset(mName, 0, sizeof(mName));
```

Comment:
If we don't use SetName anymore for this, shouldn't we set mNameLength to 0 here? Or shouldn't we get rid of mNameLenght altogether?

Reply:
you are right. Reverted the change - this was from when I was trying to get the memcmp-based equality working.

mNameLength should probably go away, but then we have to guarantee null terminator space which existing logic may not ... so I left it alone to not increase blast radius. However for the future I think it would be useful: space will be at worst the same due to padding and maybe we save some space and logic if we always store null-terminated.